client/public/employeePage/employee.js

```diff
@@ -0,0 +1,90 @@
+const addHomeAddressForm = document.getElementById("addHomeAddressForm");
+const vacationRequestForm = document.getElementById("vacationRequestForm")
+let tbody=document.getElementById("tbody");
+let messages=document.getElementById("messages");
+
+Date.prototype.toDateInputValue = (function() {
```

Comment:
why change the Date object? please use common packages such as [day.js](https://day.js.org/)